string task_id
GraspTarget[] grasp_targets
